pkg/k8shandler/certificates.go

```diff
@@ -150,12 +150,13 @@ func GenerateCertificates(namespace, scriptsDir, logStoreName, workDir string) (
 }
 
 func RunCertificatesScript(namespace, logStoreName, workDir, script string) (err error) {
-	logger.Debugf("Running script '%s %s %s %s'", script, workDir, namespace, logStoreName)
+	log.V(3).Info("Running script '%s %s %s %s'", "script", script, "workDir", workDir, "namespace", namespace, "logStoreName", logStoreName)
 	cmd := exec.Command(script, workDir, namespace, logStoreName)
 	result, err := cmd.Output()
-	if logger.IsDebugEnabled() {
-		logger.Debugf("cert_generation output: %s", string(result))
-		logger.Debugf("err: %v", err)
+
+	log.V(3).Info("cert_generation output", "output", string(result))
```

Comment:
It's worth figuring out if there is a way to query the verbocity before logging.  This will always  convert the result to a string regardless of the log level which could be an expensive copy operation

Reply:
Good catch, will try to found the way for checking verbosity settings. But looks like it not available in current logger API.

Reply:
thx, @blockloop